models/enterprise_sonic/snmp/merged_example_01.txt

```diff
@@ -1,31 +1,29 @@
 # Using merged
-
-# Before state:
-# -------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# Before state:
+#---------------
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# show running-configuration | grep snmp
+# (no snmp-server configuration present)
 #
 
-- name: Merge a single SNMP community
-  sonic_snmp:
-    config:
-      - community: "public123"
-        access: "ro"
-    state: merged
+# - name: Merge snmp-server configuration
+#   sonic_snmp:
+#       config:
+#           users:
+#               - name: user1
+#                 auth: 
+#                   md5_key: md5
+#                 priv:
+#                   aes_key: aes-128
+#                 encrypted: True
```

Comment:
This is not an encrypted key. This would fail. Set "encrypted: to "false" here (uncapitalized).

Note: On a GET or "show" after configuring this, an encrypted equivalent of this value will be returned. That same encrypted value could be used with a PATCH (merged) playbook as the key with "encrypted" set to "true".

Reply:
The changes made to address this comment are not correct, but I have posted a new comment to explain why. This "Issue" can be marked "Resolved".